PURL-TYPES.rst

```diff
@@ -1,588 +1,75 @@
-Package URL Type definitions
+Package-URL Type definitions
 ============================
 
 Each package manager, platform, type, or ecosystem has its own conventions and
 protocols to identify, locate, and provision software packages.
 
-The package **type** is the component of a package URL that is used to capture
+The package **type** is the component of a Package-URL that is used to capture
 this information with a short string such as ``maven``, ``npm``, ``nuget``, ``gem``,
 ``pypi``, etc.
 
-
-These are known ``purl`` package type definitions.
-
-Known ``purl`` type definitions are formalized here independent of the core
-Package URL specification. See also a candidate list further down.
+These are known ``PURL`` package type definitions.
```

Comment:
```suggestion
These are ``PURL`` package type definitions.
```
The spec should be clear that all types are defined here. The word `known` implies there may be other `unknown` types outside of those in the spec.

Reply:
During our meetings we agreed to use the term "registered" to denote PURL Types with a purltype-definition.json file included in the index. We will need to update or remove known in many places after this PR is merged.

Reply:
Gotcha. No worries on this suggestion then 👍

Reply:
Posted as a post merge todo in:
- https://github.com/package-url/purl-spec/issues/539